packages/webapp/src/pages/election/round-video-upload.tsx

```diff
@@ -326,10 +257,10 @@ const Header = ({
     roundEndTime,
 }: HeaderProps) => {
     const subText =
-        roundStartTime && roundEndTime
-            ? `${roundStartTime.format("LT")} - ${roundEndTime.format("LT z")}`
-            : winner && isValidDelegate(winner.account)
+        winner && isValidDelegate(winner.account)
             ? `Delegate elect: ${winner.name}`
```

Comment:
suggestion: remove the word "elect" here, since this can appear in the context of a completed election where they're officially a delegate now. ("delegate election" or "president elect" or "congressman elect" usually refer to someone who has been elected but has not yet taken office.)